packages/core-data/src/private-selectors.ts

```diff
@@ -151,7 +151,6 @@ export const getHomePage = createRegistrySelector( ( select ) =>
 			return { postType: 'wp_template', postId: frontPageTemplateId };
 		},
 		( state ) => [
-			// @ts-expect-error
```

Comment:
When I made `key` in `GetEntityRecord` optional, this line errored as an "Unused '@ts-expect-error' directive.", so I removed it and everything seems good when building 🤞